src/Util/PhpServicesCreator.php

```diff
@@ -0,0 +1,574 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+use PhpParser\Builder\Param;
+use PhpParser\BuilderFactory;
+use PhpParser\Node;
+use PhpParser\PrettyPrinter\Standard;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Yaml\Parser;
+use Symfony\Component\Yaml\Tag\TaggedValue;
+
+class PhpServicesCreator
```

Comment:
I think you can put your `@author` here ;)

Reply:
Done.